src/picongpu/include/plugins/output/images/LiveViewClient.hpp

```diff
@@ -76,7 +85,7 @@ namespace picongpu
     inline void LiveViewClient::operator() < DataBox<PitchedBox<float3_X, DIM2 > > >(
                                                                                    const DataBox<PitchedBox<float3_X, DIM2 > > data,
```

Comment:
The current coding style not define if we indent only or align function parameters!

But I will change this line.